src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -109,8 +116,16 @@ public HTTPJwtAuthenticator(final Settings settings, final Path configPath) {
         jwtHeaderName = settings.get("jwt_header", HttpHeaders.AUTHORIZATION);
         isDefaultAuthHeader = HttpHeaders.AUTHORIZATION.equalsIgnoreCase(jwtHeaderName);
         rolesKey = settings.get("roles_key");
+        jsonRolesPath = settings.get("roles_path");
```

Comment:
Is it necessary to support both `roles_path` and `roles_key`? Can one of them be deprecated? Does `roles_path` provide a superset of what `roles_key` provide?

Reply:
Actually, `roles_path` does provide a superset of what `roles_key` provide. IMO it can be deprecated. What do you think @palashhedau ?

Reply:
It can be deprecated. But let it be for now for the existing customers.

Reply:
This can be in a new PR.